packages/slate-types/src/nodes/VariableNode.ts

```diff
@@ -5,6 +5,7 @@ export const VARIABLE_NODE_TYPE = 'variable';
 
 export interface VariableNode extends ElementNode {
     type: typeof VARIABLE_NODE_TYPE;
+    fallback?: string | null;
```

Comment:
@kudlajz by the way, Slate does not allow `null` as a node property value. Setting a property to null with `Transforms.setNodes()` will remove it from the object.